peer/peer.go

```diff
@@ -7,11 +7,10 @@ import (
 
 // Peer reperesents a GlusterD
 type Peer struct {
-	ID        uuid.UUID `json:"id"`
-	Name      string    `json:"name"`
-	Addresses []string  `json:"addresses"`
-	Client    bool      `json:"client"`
-	MemberID  uint64    `json:"memberID"`
+	ID       uuid.UUID `json:"id"`
+	Name     string    `json:"name"`
+	Address  string    `json:"address"`
```

Comment:
It's okay if the peer add ReST API had just one initial address. But a peer could have multiple addresses. While we don't yet have a way to add newer addresses, we can atleast be ready here for when we have it.